packages/cfd/src/Containers/props.types.ts

```diff
@@ -56,7 +56,6 @@ export type TMT5AccountOpeningRealFinancialStpModal = {
     enableApp: () => void;
     disableApp: () => void;
     toggleCFDVerificationModal: () => void;
```

Comment:
cant we remove this "toggleCFDVerificationModal". can you check this

Reply:
we can remove this as well doesn't break anything